### PR TITLE
Add release notes for SoCo 0.11

### PR DESCRIPTION
**Pull request by stefankoegl**
I've collected the release notes for the upcoming 0.11 release (see #278 and #261).

Please let me know if I forgot anything, or if my wording needs to be improved.

/cc all contributors to the release @simonalpha @trollkarlen @lawrenceakka @dajobe @robwebset @gutworth @poirier @michaelotto @hildroid @KennethNielsen

### CODE REVIEW

**Comment by KennethNielsen** [comment]
@stefankoegl I just merged #225, which was planned for this release and has been ready for while. I was unable to tell whether it passed travis tests since it hadn't added the dots for the last commit, but otherwise I will help in fixing any issues. I also asked dajobe to add a line here about the pull.

File: doc/releases/0.11.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+* It is now possible to trigger an update of the music library
+  (`#286 <https://github.com/SoCo/SoCo/pull/286>`_)
+
+* The event listener port is now configurable
+  (`#288 <https://github.com/SoCo/SoCo/pull/288>`_)
+
+* Methods that can only be executed on master speakers will now raise a
+  ``SoCoSlaveException`` (`#296 <https://github.com/SoCo/SoCo/pull/296>`_)
+
+* An example has been added that shows how to play local files by settings up a
```
Comment (KennethNielsen) [inline doc/releases/0.11.rst]: setting[no S]

**Comment by KennethNielsen** [comment]
@stefankoegl Ah I see you already added a line about #225. Otherwise, just a single inline comment, nice work.

**Comment by stefankoegl** [comment]
Fixed. Thanks for the review :)

**Pull request merged**